samples/test/rollout.js

```diff
@@ -162,10 +162,16 @@ describe('Game Server Rollout Test', () => {
       ),
     };
 
-    const [rollout] = await deploymentClient.getGameServerDeploymentRollout(
-      request
-    );
-    assert.strictEqual(rollout.gameServerConfigOverrides.length, 0);
+    let rollout;
+    try {
+      [rollout] = await deploymentClient.getGameServerDeploymentRollout(
+        request
+      );
+    } catch (err) {
+      if (!err.message.includes(/The service is currently unavailable/)) {
+        assert.strictEqual(rollout.gameServerConfigOverrides.length, 0);
```

Comment:
In this case `assert.strictEqual` will only be executed if the API call threw an exception, but it won't check anything in case if the call was successful and did not throw.